components/formats-gpl/matlab/bfCheckJavaPath.m

```diff
@@ -50,11 +50,12 @@
 % Check if a Bio-Formats JAR file is in the Java class path
 % Can be in either static or dynamic Java class path
 jPath = javaclasspath('-all');
-bfJarFiles = {'bioformats_package.jar', 'loci_tools.jar'};
+jars = cell2mat (cell2mat (regexp (jPath, '[\\/]{1}([^/\\]+)\.jar$', 'tokens')));
```

Comment:
Unfortunately this change fails to run in MATLAB - see error messages in https://ci.openmicroscopy.org/job/BIOFORMATS-5.1-merge-matlab/924/